src/perl.c

```diff
@@ -1008,11 +1011,24 @@ static int pplugin_call_all (pTHX_ int type, ...)
 {
 	int retvals = 0;
 
+	_Bool old_running;
 	va_list ap;
 	int ret = 0;
 
 	dSP;
 
+	c_ithread_t *t = (c_ithread_t *)pthread_getspecific(perl_thr_key);
+	if (t == NULL) /* thread destroyed ( c_ithread_destroy*() -> log_debug() ) */
```

Comment:
Please write "(c_ithread_destroy...)" instead of "( c_ithread_destroy… )" (that is, drop the space after "(" and before ")".